### DISCUSSION TITLE
Integrate with Qt IDE?

### DISCUSSION THREAD

**Discussion by electron271** [microsoft/vcpkg#17289] [Ideas]
Possibly integrate with Qt?

**Comment by autoantwort** [comment]
Like here: https://www.qt.io/blog/qt-creator-cmake-package-manager-auto-setup  
Or what did you expect?

**Reply by electron271** [reply]
QMake integration possibly

**Reply by electron271** [reply]
If it can happen

**Reply by autoantwort** [reply]
Maybe someone can write a `vcpkg.pri` file that you can include that automatically calls `vcpkg install` and add the result to `INCLUDEPATH` and `LIBS`. That should be rather easy. The only hard thing is selecting the right triplet. Afaik there is currently no way to detect the target architecture (but you can use the host architecture)
